Framework/Events.php

```diff
@@ -24,10 +25,16 @@
 require_once 'Modules/quote/Model/Quote.php';
 require_once 'Modules/services/Model/SeService.php';
 
+require_once 'Modules/invoices/Model/GlobalInvoice.php';
+
+require_once 'Modules/booking/Model/BookingInvoice.php';
+require_once 'Modules/services/Model/ServicesInvoice.php';
+
```

Comment:
missing import: require_once 'Modules/booking/Model/BookinginvoiceTranslator.php';

Reply:
not needed here, certainly elsewhere, i gonna look

Reply:
ah confusing with translators.... ok

Reply:
in bookinginvoice.php